### PR TITLE
App Sec OOTB Rules

### PR DESCRIPTION
**Pull request by sarina-dd**
### What does this PR do?
- Updates the `security_rules.py` file to pull in new app sec rules;
- Updates Cloud Configuration and Infrastructure Configuration to Posture Management (Cloud) and Posture Management (Infra) for product context
- Updates pull configs for the build
- Updates content on OOTB rules page for product clarity

### Motivation
Docs OKR / Jira

### Preview
https://docs-staging.datadoghq.com/sarina/app-sec-ootb-rules/security_platform/default_rules/

### QA steps

On the OOTB page:

1. Click all product-named buttons to ensure they work
2. Under `application security`, click on individual rules to make sure they are populating

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by davidejones — APPROVED** [review]
I did notice that the left nav `Security Platform -> Cloud SIEM -> OOTB Rules` link might need updating to the new anchor category. Other than that the changes look good here.

**Comment by arnaudbreton** [comment]
image
The display of the classification looks odd https://docs-staging.datadoghq.com/sarina/app-sec-ootb-rules/security_platform/default_rules/appsec-log4j-attack-attempt/

**Review by arnaudbreton — APPROVED** [review]

**Comment by arnaudbreton** [comment]
The goal is to have this under "Application Security" and as "OOTB Detection rules" right?

image

**Comment by arnaudbreton** [comment]
image
This dropdown doesn't have much value in our context

File: content/en/security_platform/default_rules/_index.md [hunk 1]
```diff
@@ -9,6 +9,10 @@ aliases:
 disable_sidebar: true
 ---
 
-Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so that you can immediately take steps to remediate. Datadog continuously develops new default detection rules, which are automatically imported into your account.
+Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account or with the Agent. For more information, see the Detection Rules documentation.
```
Comment (arnaudbreton) [inline content/en/security_platform/default_rules/_index.md:12]: Agent + library (AppSec)

File: content/en/security_platform/default_rules/_index.md [hunk 2]
```diff
@@ -9,6 +9,10 @@ aliases:
...
 
-Filter by **Logs Detection** to see the Cloud SIEM detection rules, **Workload Security** to see Cloud Security Workload detection rules, **Cloud Configuration** or **Infrastructure Configuration** to see the Cloud Security Posture detection rules.
+Click on the buttons below to filter by different parts of the Datadog Security Platform. OOTB rules are available for [Cloud SIEM][1], [Posture Management][2], which is divided into cloud or infrastructure configuration, [Workload Security][3], and Application Security.
```
Comment (arnaudbreton) [inline content/en/security_platform/default_rules/_index.md:14]: Miss link to AppSec

**Comment by sarina-dd** [comment]
> The goal is to have this under "Application Security" and as "OOTB Detection rules" right?
> 
> 
> 
> image
> 
> 

Correct, however the nav is causing issues here because it's jumping back up to the Cloud SIEM section when you click OOTB rules from the Cloud SIEM section. 

👋 @davidejones Do you have any suggestions here to resolve the nav? Should we duplicate the index page into each section to resolve?

**Comment by sarina-dd** [comment]
> image
> 
> This dropdown doesn't have much value in our context
> 
> 
> 
> One usage of it could be to split "Attack attempts" from "Vulnerability triggered"

Good point. This requires add't configuration of the .py file to split, so let's leave for now as. I created a separate Jira issue for us to iterate on this after publishing.

File: content/en/security_platform/default_rules/_index.md (hunk 1 above)
Comment (sarina-dd) [inline content/en/security_platform/default_rules/_index.md:12]:
```suggestion
Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and with the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```

File: content/en/security_platform/default_rules/_index.md (hunk 2 above)
Comment (sarina-dd) [reply content/en/security_platform/default_rules/_index.md:14]: Intentionally removed as we are publishing this page first. Will add the link once Kari's branch is published with the docs.

File: content/en/security_platform/default_rules/_index.md [hunk 3]
```diff
@@ -9,6 +9,10 @@ aliases:
 disable_sidebar: true
 ---
 
-Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so that you can immediately take steps to remediate. Datadog continuously develops new default detection rules, which are automatically imported into your account.
+Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and with the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```
Comment (kayayarai) [inline content/en/security_platform/default_rules/_index.md:12]:
```suggestion
Datadog provides out-of-the-box (OOTB) detection rules to flag attacker techniques and potential misconfigurations so you can immediately take steps to remediate. Datadog continuously develops new default rules, which are automatically imported into your account, your Application Security library, and the Agent, depending on your configuration. For more information, see the Detection Rules documentation.
```

Comment (kayayarai) [reply content/en/security_platform/default_rules/_index.md:12]: Also, is there a link for "Detection Rules documentation"?

**Review by kayayarai — CHANGES_REQUESTED** [review]

File: content/en/security_platform/default_rules/_index.md (hunk 3 above)
Comment (sarina-dd) [reply content/en/security_platform/default_rules/_index.md:12]: Why yes, there is! ✨  Added.

File: content/en/security_platform/default_rules/_index.md [hunk 4]
```diff
@@ -9,6 +9,10 @@ aliases:
...
 
-Filter by **Logs Detection** to see the Cloud SIEM detection rules, **Workload Security** to see Cloud Security Workload detection rules, **Cloud Configuration** or **Infrastructure Configuration** to see the Cloud Security Posture detection rules.
+Click on the buttons below to filter by different parts of the Datadog Security Platform. OOTB rules are available for [Cloud SIEM][1], [Posture Management][2], which is divided into cloud or infrastructure configuration, [Workload Security][3], and Application Security.
+
+[1]: /security_platform/cloud_siem/
+[2]: /security_platform/cspm/
+[3]: /security_platform/cloud_workload_security/
```
Comment (sarina-dd) [inline content/en/security_platform/default_rules/_index.md:18]:
```suggestion
[3]: /security_platform/cloud_workload_security/
[4]: /security_platform/detection_rules/
```

**Review by kayayarai — APPROVED** [review]

**Comment by arnaudbreton** [comment]
For Thursday release, the title will be "OOTB rules" right? Vs "OOTB Detection Rules"

**Pull request merged**